Cargo.toml

```diff
@@ -4,13 +4,15 @@ members = [
     "crates/composer_api",
     "crates/dtrace-sys",
     "crates/test_probe",
+    "crates/ptrace_probe",
 ]
 
 # Specify a subset of member crates that compile on all supported architectures.
 default-members = [
     "crates/composer_api",
     "crates/composer",
     "crates/test_probe",
+    "crates/ptrace_probe",
```

Comment:
@goodhoko / @skywhale this does compile on a Mac, right?

Reply:
It doesn't. I only noticed this after merging. I do have a WIP branch where I remove this line and adapt the CI to  run both on linux and macOS to lint/test all crates.